src/java.base/share/classes/java/lang/ClassValue.java

```diff
@@ -34,6 +40,10 @@
 import static java.lang.ClassValue.ClassValueMap.probeHomeLocation;
 import static java.lang.ClassValue.ClassValueMap.probeBackupLocations;
 
+/*[IF CRIU_SUPPORT]*/
+import openj9.internal.criu.NotCheckpointSafe;
+/*[ENDIF] CRIU_SUPPORT*/
```

Comment:
Please leave a space after the repeated expression (like we do for C/C++ code):
```
/*[ENDIF] CRIU_SUPPORT */
```

Reply:
Added a space.